wordpress-mobile/WordPress-Android
# Add UI design to the reusable Jetpack Static Poster

Resolves #18115

This PR:

- ★ Implements the UI of the Jetpack Static Poster screen
- ⊕ Adds event handling and tracking

## To Test

1. Go to `Me` → `App Settings` → `Debug settings`
2. Scroll to the `Features in development` section
3. Tap on `jp_removal_static_posters`
4. Go to either: Stats, Reader or Notifications and:

- **Verify** start is tracked
- **Verify** clicks on the primary button are handled and tracked
- **Verify** clicks on the secondary button are handled and tracked
- **Verify** the UI matches the design
- **Verify** the animated icon above the title adapts to LTR/RTL
- **Verify** pluralisation is correctly applied to the feature name in the title (`has` vs `have`)

### Previews

| App | Design |
| - | - |
| <img width="315" src="https://user-images.githubusercontent.com/4588074/225930981-1fa59ff1-7fb2-4864-b62b-5328db27d0cd.png"> | <img width="315" src="https://user-images.githubusercontent.com/4588074/225930901-435d3b7e-0c0a-408c-a217-885128c15cc8.png"> |

<details><summary>📹 <b>Recording</b> — Animations, Rtl & Pluralisation</summary>

<video src="https://user-images.githubusercontent.com/4588074/225978814-fe04db59-9a38-4922-b8f5-6dca76c17c09.mp4" style="max-width: 315px;"></video>
</details>

## Regression Notes

1. Potential unintended areas of impact
   None - the impacted functionality concerns a new feature that was integrated in earlier PRs.
   There might be low-chance regressions caused by extending the existing `PrimaryButton` and `SecondaryButton` UI components, by testing the Jetpack Migration Flow UI any possible regressions will be surfaced. This is expected to be the case, as the functionality added in this PR would start that flow.

7. What I did to test those areas of impact (or what existing automated tests I relied on)
   Manual testing

8. What automated tests I added (or what prevented me from doing so)
   N/a - only UI changes for a new screen

PR submission checklist:
